gamestonk_terminal/stocks/fundamental_analysis/fa_controller.py

```diff
@@ -460,23 +457,34 @@ def call_divs(self, other_args: List[str]):
             add_help=False,
             formatter_class=argparse.ArgumentDefaultsHelpFormatter,
             prog="divs",
-            description="Get historical dividends for company",
+            description="Historical dividends for a company",
         )
         parser.add_argument(
             "-n",
             "--num",
             dest="num",
             type=check_positive,
-            default=12,
+            default=20,
             help="Number of previous dividends to show",
         )
+        parser.add_argument(
+            "-p",
+            "--plot",
+            dest="plot",
+            default=True,
```

Comment:
I think you want default false here

Reply:
IMO, it's more telling to see the visual, gives a clear signal for strength of a dividend paying company.